extensions/integration-tests/test/stressutils.test.ts

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+import 'mocha';
+import { runOnCodeLoad } from '../src/fmkUtils';
+import { stressify, sleep, bear } from '../src/stressutils'; //, bear, sleep
+import assert = require('assert');
+
+class StressifyTester {
+	static dop: number = 5;
+	static iter: number = 4;
+
+	i: number = 0;
+	@runOnCodeLoad(StressifyTester.prototype.setup)
+	setup() {
+		process.env.SuiteType = 'Stress';
+		console.log(`environment variable SuiteType set to ${process.env.SuiteType}`);
+	}
+
+	@stressify({ dop: StressifyTester.dop, iterations: StressifyTester.iter })
+	async basicTest(arg: string) {
+		bear();	// yield to other operations.
+		await sleep(50); // sleep for 100 ms without spinning
+		//console.log(`basicTest() called for i=${this.i}`);
+		this.i++;
+	}
+}
+
+suite('StressUtils unit tests', function () {
+	setup(function () {
+		this.Tester = new StressifyTester();
+	});
+	test('basicTest', async function () {
```

Comment:
Would suggest adding tests for verifying that passed/failed counts are accurate as well

Reply:
Agreed - already on my todo list.